src/contrib/custom_op/json/custom_test_op.json

```diff
@@ -0,0 +1,15 @@
+{
+    "ops": [
+        {
+            "op": "CustomOpTest",
```

Comment:
Shall we require users to implement the identifier field in the custom_op? We can also implement an identifier generator for generic ops, which could avoids the users to implement the identifier for custom ops.